### ISSUE TITLE
Uncaught TypeError: Cannot read properties of undefined (reading 'initOAuth')

### ISSUE THREAD

**Issue by akanass** [nestjs/swagger#1970]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

`swagger-ui-init.js:2424 Uncaught TypeError: Cannot read properties of undefined (reading 'initOAuth') at window.onload (swagger-ui-init.js:2424:21)`

```js
if (customOptions.initOAuth) {
  ui.initOAuth(customOptions.initOAuth)
}
```

but at the line `2402` `customOptions` is never initialised if `options.customOptions` is undefined.

```js
let customOptions = options.customOptions
```

### Minimum reproduction code

[repository](https://github.com/akanass/nestjs-swagger)

### Steps to reproduce

Follow the readme

### Expected behavior

not having the error in the console

### Package version

6.0.3

### NestJS version

9.0.3

### In which operating systems have you tested?

- [X] macOS
- [ ] Windows
- [X] Linux

**Comment by flamewow** [comment]
@akanass 
Please provide a [minimum reproduction repository](https://minimum-reproduction.wtf/) (Git repository/[StackBlitz](https://stackblitz.com/fork/github/nestjs/typescript-starter)/[CodeSandbox](https://codesandbox.io/s/github/nestjs/typescript-starter/tree/master) project).

**Comment by akanass** [comment]
@flamewow ok I'm on it

**Comment by kamilmysliwiec** [comment]
Fixed in 6.0.4
